Unix/sock/sock.c

```diff
@@ -361,6 +361,7 @@ MI_Result Sock_Connect(
 
     r = connect(self, &addr->u.sock_addr, (size_t)addr->sock_addr_size);
     connectErrno = errno;
+    (void) connectErrno;
```

Comment:
Doesn't this need to be #ifdef'ed for DEBUG or something? I thought that's what we discussed yesterday ...